application/src/main/java/org/togetherjava/tjbot/config/CakeDayConfig.java

```diff
@@ -0,0 +1,10 @@
+package org.togetherjava.tjbot.config;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Configuration record for the Cake Day feature.
+ */
+public record CakeDayConfig(
+        @JsonProperty(value = "rolePattern", required = true) String rolePattern) {
+}
```

Comment:
Add `Objects.requireNonNull` checks in constructor

Reply:
That is now done! 🙌